docs/catalog/triggers/kubernetes.rst

```diff
@@ -46,3 +46,25 @@ The following wildcard triggers will fire for any supported Kubernetes resource:
 * on_kubernetes_any_resource_update
 * on_kubernetes_any_resource_delete
 * on_kubernetes_any_resource_all_changes
+
+Supported filters
+--------------------
+
+All Kubernetes triggers support the following filters:
+
+* ``name_prefix`` - Name prefix to match resources.
+* ``namespace_prefix`` - Namespace prefix to match resources.
+* ``labels_selector`` - Match resources with these labels. The format is: ``label1=value1,label2=value2``. If more than one labels is provided, **all** needs to match.
+
+For example:
```

Comment:
Can we put the example at the top of the page along with one sentence on the motivation?

I think we're jumping in too fast into the fine details without explaining the motivation. We can't assume that people read through the docs and reached this page in the order we expect.

See for example, what I did at https://docs.robusta.dev/master/catalog/triggers/prometheus.html

Reply:
good idea. Fixing